From 93678f715c4b809d3856957b53d3452be298de3a Mon Sep 17 00:00:00 2001
From: Nicola Fontana <ntd@entidi.it>
Date: Sat, 14 Mar 2015 23:28:35 +0100
Subject: [PATCH 02/11] Use pkg-config to get python2 flags

The old way of using python2-config is cross-unfriendly because runs a
python program that returns paths suitable for the host platform, not
for the build platform.
---
 m4/python.m4 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/m4/python.m4 b/m4/python.m4
index ed5559d..6b4bd08 100644
--- a/m4/python.m4
+++ b/m4/python.m4
@@ -43,7 +43,7 @@ AC_DEFUN([AM_CHECK_PYTHON_HEADERS],
 [AC_REQUIRE([AM_PATH_PYTHON])
 AC_MSG_CHECKING(for headers required to compile python extensions)
 dnl deduce PYTHON_INCLUDES
-PYTHON_INCLUDES=`$PYTHON-config --includes`
+PYTHON_INCLUDES=`$PKG_CONFIG --cflags python2`
 AC_SUBST(PYTHON_INCLUDES)
 dnl check if the headers exist:
 save_CPPFLAGS="$CPPFLAGS"
@@ -64,7 +64,7 @@ AC_DEFUN([AM_CHECK_PYTHON_LIBS],
 AC_MSG_CHECKING(for libraries required to link against libpython)
 dnl deduce PYTHON_LIBS
 if test "x$PYTHON_LIBS" = x; then
-	PYTHON_LIBS=`$PYTHON-config --ldflags --libs`
+	PYTHON_LIBS=`$PKG_CONFIG --libs python2`
 fi
 AC_SUBST(PYTHON_LIBS)
 dnl check if libpython exist:
-- 
2.7.0

